dev/com.ibm.ws.security.acme/resources/com/ibm/ws/security/acme/resources/AcmeMessages.nlsprops

```diff
@@ -206,3 +206,19 @@ CWPKI2044E.useraction=Ensure that all the certificates in the certificate chain
 CWPKI2045W=CWPKI2045W: The certificate with {0} serial number that is signed by the ACME certificate authority at the {1} URI is not valid until {2}.
 CWPKI2045W.explanation=The valid period on the certificate is in the future. SSL/TLS requests fail until the current date and time are within the range that is specified by the valid period on the certificate.
 CWPKI2045W.useraction=Update the local time on the server if the time is incorrect.
+
+CWPKI2052W=CWPKI2052W: The ACME Service failed to automatically renew the certificate with {0} serial number. The certificate will expire on {1}. The renew request error is ''{2}''. The request is scheduled to try again in {3}.
+CWPKI2052W.explanation=The ACME Service tried to renew a certificate but encountered an error. The ACME Service will continue to request a new certificate until it is successful.
+CWPKI2052W.useraction=Review the error message for details on the failure.
+
+CWPKI2053E=CWPKI2053E: The ACME Service failed to automatically renew the certificate with {0} serial number. The certificate is revoked. The renew request error is ''{1}''. The request is scheduled to try again in {2}.
+CWPKI2053E.explanation=The ACME Service tried to renew a certificate but encountered an error. The ACME Service will continue to request a new certificate until it is successful. SSL/TLS requests will fail until a new certificate request is successful.
```

Comment:
**change**
The ACME Service will continue to request a new certificate until it is successful. SSL/TLS requests will fail until a new certificate request is successful.
**to**
The ACME Service continues to request a new certificate until a new certificate is issued. SSL/TLS requests fail until a new certificate request is successful.